mam/v2/mam/message.c

```diff
@@ -547,7 +545,7 @@ void mam_msg_send(mam_msg_send_context_t *ctx, mam_prng_t *prng,
   /* wrap Header */
   {
     /*  absorb tryte msgid[27]; */
-    pb3_wrap_absorb_ntrytes(&ctx->spongos, msg, msg_id);
+    pb3_absorb_external_ntrytes(&ctx->spongos, msg_id);
```

Comment:
Technically, this works. But why not add an extra argument to `mam_msg_send` - `trits_t tag` and
```
pb3_wrap_absorb_ntrytes(&ctx->spongos, &tag, msg_id);
```
?
It will make sense as `tag` is a part or message, it's just a different location of `msg_id` and `ord`.